pkgs/misc/vim-plugins/generated.nix

```diff
@@ -135,18 +135,6 @@ let
     meta.homepage = "https://github.com/vim-scripts/argtextobj.vim/";
   };
 
-  asyncomplete-vim = buildVimPluginFrom2Nix {
-    pname = "asyncomplete-vim";
-    version = "2020-08-09";
-    src = fetchFromGitHub {
-      owner = "prabirshrestha";
-      repo = "asyncomplete.vim";
-      rev = "0d4ec1805d50fd39b8b41626e0761736feb307d0";
-      sha256 = "0lcqdh51fd8lx1g02dcwrfnnx3zfm71g57nv2pmin5ap7qgh8kir";
-    };
-    meta.homepage = "https://github.com/prabirshrestha/asyncomplete.vim/";
-  };
-
```

Comment:
What is the reasoning behind removing this package?

Reply:
This was done by `update.py`. It looks like the package has a new owner on GitHub, and the `update.py` script follows redirect links. A recent PR I made for vimPlugins featured the same update.

Reply:
Hmm the package has not moved from https://github.com/prabirshrestha/asyncomplete.vim, and I did not make this change. I'll re-run the script :crossed_fingers:

Reply:
Looks like it was introduced in 2bc02f29256c0ba5c366330405502637c73c1fd1 but was not introduced properly.

Reply:
Yes that looks to be the issue. Good eye. It was not included in vim-plugin-names.

Reply:
I addressed this issue, PTAL.